modules/tinymce/src/core/main/ts/dom/ScrollIntoView.ts

```diff
@@ -117,47 +117,79 @@ const preserveWith = (editor: Editor, f: (startElement: SugarElement<Node>, endE
   editor.selection.setRng(rng);
 };
 
-const scrollToMarker = (editor: Editor, marker: MarkerInfo, viewHeight: number, alignToTop: boolean, doc?: SugarElement<Document>) => {
+const scrollToMarker = (editor: Editor, marker: MarkerInfo, viewHeight: number, alignToTop: boolean, doc?: SugarElement<Document>, topMargin?: boolean) => {
   const pos = marker.pos;
+  const viewWidth = Traverse.defaultView(doc ?? SugarElement.fromDom(editor.getDoc())).dom.innerWidth;
+  const viewportLeft = -editor.getBody().getBoundingClientRect().left;
   if (alignToTop) {
-    Scroll.to(pos.left, pos.top, doc);
+    if (topMargin) {
+      const targetOffset = viewHeight * 0.25;
```

Comment:
This value should be configurable either to a fixed value or percentage value. You could have `scrollIntoView(elm, { topMargin: '25%' })` then allow fixed values either with `px` or just regular numbers not sure we need that now but could be added later.

Reply:
The implementation has changed to a fixed value